src/TimeModeling/judiWeights.jl

```diff
@@ -82,6 +82,13 @@ adjoint(a::judiWeights{vDT}) where vDT =
 ##########################################################
 
 
+# -(judiWeights)
+function -(a::judiWeights{avDT}) where {avDT}
```

Comment:
May be missing in judiVector too would be good to fix as well.

Reply:
Gotcha!